src/akar/util.clj

```diff
@@ -0,0 +1,48 @@
+(ns akar.util
+  (:require [clojure.walk :as cw]))
+
+
```

Comment:
Please use one newline between forms. Nothing wrong with two, but we have used one everywhere else, so it's better to be consistent.

Reply:
done